index.js

```diff
@@ -0,0 +1,30 @@
+const [navList, navAdd, navContact] = document.querySelectorAll('.list-item');
+const allBooks = document.querySelector('.allBooks');
+const addBook = document.querySelector('.addBook');
+const contact = document.querySelector('.contact');
+const show = document.querySelector('.show');
+const display = document.querySelector('.display-tex');
+
+navList.addEventListener('click', () => {
+  allBooks.classList.remove('hidden');
+  show.classList.remove('hidden');
+  display.classList.remove('hidden');
+  addBook.classList.add('hidden');
+  contact.classList.add('hidden');
+});
```

Comment:
- [ ] currently, when I navigate back to `mylist` link after visiting other links,  the other item still shows in `mylist` page, kindly fix this, you can try this method as I already tried it in the code editor :point_down: 
```
navList.addEventListener('click', () => {
  addBook.classList.add('hidden');
  contact.classList.add('hidden');
  allBooks.classList.remove('hidden');
  show.classList.remove('hidden');
  display.classList.remove('hidden');
});
```add `hidden` class to other DOM content before removing class list from `display`